src/gsi_ncdiag/ncdiag_to_feedback.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import argparse
+import time
+import sys
+import os
+import shutil
+import h5py
+
+
+def main(oman, ombg, output_file, output_dir='./'):
+
+    # start timer
+    tic = record_time()
+
+    print()
+    print(f'List of files to process: {oman} {ombg}')
+    print()
+
+    # open first guess (background) departure
+    f = h5py.File(ombg, 'r')
+    # open analysis departure
+    g = h5py.File(oman, 'r')
+    # duplicate (destructive) the analysis departure
+    shutil.copyfile(oman, output_file)
+    # open output ioda in read write
+    k = h5py.File(output_file, 'r+')
+
+    # start appending data to output_file
+    f.copy(f['GsiHofXBc'], k, 'hofx0')
+    g.copy(g['GsiHofXBc'], k, 'hofx1')
```

Comment:
This should be hofx2 to be "correct", given that it is after the second (final) minimization loop

Reply:
noted